### PR TITLE
[docs] Collecting stack traces with native symbols on WebAssembly

### PR DESCRIPTION
**Pull request by lambdageek**

### CODE REVIEW

File: docs/workflow/debugging/mono/wasm-debugging.md [hunk 1]
```diff
@@ -133,3 +133,131 @@ These kinds of faults usually happen because the mono runtime has some helper fu
...
+```
+
+In order to get symbols, the user should:
+
+1. Install the `wasm-tools` workload using `dotnet workload install wasm-tools`
+2. Set these additional properties in their `.csproj` file:
+
+   ```xml
+     <!-- Builds a dotnet.wasm with debug symbols preserved -->
+     <PropertyGroup Condition="true">
```
Comment (radical) [inline docs/workflow/debugging/mono/wasm-debugging.md:208]:
```suggestion
     <PropertyGroup>
```

File: docs/workflow/debugging/mono/wasm-debugging.md [hunk 2]
```diff
@@ -133,3 +133,131 @@ These kinds of faults usually happen because the mono runtime has some helper fu
 never meant to be reached, i.e. `no_gsharedvt_in_wrapper` or `no_llvmonly_interp_method_pointer`.
 These functions are used as placeholders for function pointers with different signatures, so
 if they do end up being called due to a bug, a signature mismatch error happens.
+
+# Collecting stack traces with symbols in Blazor
+
+If a customer has a native crash in a .NET 6 Blazor app or another WebAssembly
```
Comment (lewing) [inline docs/workflow/debugging/mono/wasm-debugging.md:139]:
```suggestion
When debugging a native crash in a .NET 6 Blazor app or another WebAssembly
```
or perhaps `When examining`

**Review by lewing — APPROVED** [review]
Thanks! @lambdageek

**Review by radical — APPROVED** [review]

**Pull request merged**